solution/backend/cmcs_regulations/context_processors.py

```diff
@@ -20,6 +20,12 @@ def survey_url(request):
     }
 
 
+def signup_url(request):
+    return {
+        "SIGNUP_URL": settings.SIGNUP_URL
+    }
+
+
```

Comment:
suprised the linter didnt catch this but one blank line after  a function.  So remove line 27 and also remove line 22.

Reply:
The linting rule is two lines between classes, not one: https://www.flake8rules.com/rules/E305.html